webhook-subscription-2022.md

```diff
@@ -1,22 +1,30 @@
-# Solid Webhook Notifications
+# WebHookSubscription2022
 
-Status: Proposal
+## Abstract
+The [Solid Notification Protocol](https://solid.github.io/notifications/protocol) defines a set of interaction patterns for agents to establish subscriptions to resources in a Solid Storage.
 
-## Introduction
+This specification defines a subscription type that applies these patterns to WebHooks.
 
-The Solid Notifications Specification allows for many methods by which a client is notified of actions being performed on a resource. One such method of notification is webhooks. Webhooks allow servers to register a target URL with a Solid Pod. When that server needs to be notified of an action, that target URL will be called by the Solid Pod.
+## Status of This Document
+TODO
+
+## 1. Introduction
+_This section is non-normative._
+
+The [Solid Notifications Protocol](https://solid.github.io/notifications/protocol) allows for many methods by which a client is notified of actions being performed on a resource. One such method of notification is webhooks. Webhooks allow servers to register a target URL with a Solid Pod. When that server needs to be notified of an action, that target URL will be called by the Solid Pod.
```

Comment:
This is a non-normative section, so this is just a suggestion for consideration. Generally, with specification documents, the term "Solid Storage" is more precise than "Solid Pod". The informal "Solid Pod" is great in colloquial contexts, but for these documents, I think "Solid Storage" would be preferred.